### ISSUE TITLE
Increase recognition intervall

### ISSUE THREAD

**Issue by tommen23** [paviro/MMM-Facial-Recognition#3]
Hi
Would it be possible to increase the recognition intervall when a PIR sensor senses someone in front of the mirror and put the camera on hold/sleep at other times?

**Comment by paviro** [comment]
Yes, it was in the v1 version just have not ported it yet but it will come 😊
